### DISCUSSION TITLE
Forever Page Loading

### DISCUSSION THREAD

**Discussion by starmomou** [Budibase/budibase#3248] [Help] [answered]
Hi everyone,
I'm new to Budibase and only have a little knowledge of development. I just self hosted Budibase using docker compose on my MacBook Air. I can create row and column very easily. However, When I go to *Design* -> *Layouts/Screens*. The Page keeps loading and it's still loading now. Why would this happen? Thanks.
<img width="1440" alt="螢幕截圖 2021-11-04 下午3 45 12" src="https://user-images.githubusercontent.com/73826858/140278184-41c1e917-1d5f-42b2-9069-bdbe758a967e.png">

**Comment by mjashanks** [comment, marked as answer]
Hey @starmomou - are you using Safari? If so, we have an open issue about this: https://github.com/Budibase/budibase/issues/2945

It's a bug in Budibase when using Safari. For now, Budibase will work fine on Chrome, Firefox and Edge. We're working on a fix for safari!

**Reply by starmomou** [reply]
@mjashanks It works perfectly on Chrome! Thanks

**Reply by jlangree** [reply]
@mjashanks  - I'm having the same issue on Firefox. It seems to work in Chrome, and it _used_ to work in Firefox for me. Any insight?

**Reply by mjashanks** [reply]
@jlangree I am not seeing this issue on Firefox or Chrome.

Are you running on Budibase cloud? If so, email your subdomain to support@budibase.com (e.g. `https://subdomain.budibase.app`)
Self hosting? What version are you running on (you can find this in the `Updates` section).

When did this start happening?

**Reply by jlangree** [reply]
Yes, Budibase cloud. It's been updated to 1.0.26, but still does the same thing. Thanks- I'll email support.

There are also all sorts of other issues. Yesterday I lost hours worth of work because it wasn't saving the "design" section updates. And the REST API suddenly always says "Query error: worker call timed out!" Nothing in the query was changed. Budibase is whitelisted. It just suddenly stopped working. It used to work just fine.
